### PR TITLE
chore!: Normalize repository, dropping node <10.13 support

### PR DESCRIPTION
**Pull request by sttk**
This PR updates this project:

- Updates dependency versions
- Introduces GitHub Actions
- Adds, changes and deletes some project files

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2015 Blaine Bublitz, Eric Schoffstall and other contributors
+Copyright (c) 2015, 2021 Blaine Bublitz, Eric Schoffstall and other contributors
```
Comment (phated) [inline LICENSE:3]:
```suggestion
Copyright (c) 2015-2019, 2021 Blaine Bublitz <blaine.bublitz@gmail.com> and Eric Schoffstall <yo@contra.io>
```
Should always have the emails

File: README.md [hunk 2]
```diff
@@ -91,31 +91,24 @@ Errors can be caused by:
...
+[ci-image]: https://img.shields.io/github/workflow/status/gulpjs/async-done/dev?style=flat-square
 
 [coveralls-url]: https://coveralls.io/r/gulpjs/async-done
-[coveralls-image]: https://img.shields.io/coveralls/gulpjs/async-done/master.svg
-
-[gitter-url]: https://gitter.im/gulpjs/gulp
-[gitter-image]: https://badges.gitter.im/gulpjs/gulp.svg
+[coveralls-image]: https://img.shields.io/coveralls/gulpjs/async-done/master.svg?style=flat-square
 
 [end-of-stream]: https://www.npmjs.com/package/end-of-stream
```
Comment (phated) [inline README.md:105]:
```suggestion
[coveralls-image]: https://img.shields.io/coveralls/gulpjs/async-done/master.svg?style=flat-square
<!-- prettier-ignore-end -->

<!-- prettier-ignore-start -->
[end-of-stream]: https://www.npmjs.com/package/end-of-stream
```
We should separate these for easier updating with `update-template` later.

File: package.json [hunk 3]
```diff
@@ -25,29 +25,37 @@
...
-    "stream-exhaust": "^1.0.1"
+    "end-of-stream": "^1.4.4",
+    "once": "^1.4.0",
+    "process-nextick-args": "^2.0.1",
+    "stream-exhaust": "^1.0.2"
   },
   "devDependencies": {
-    "@types/node": "^9.3.0",
+    "@types/node": "^16.11.7",
     "coveralls": "github:phated/node-coveralls#2.x",
```
Comment (phated) [inline package.json:38]: I believe we should remove this coveralls dep now
```suggestion
    "@types/node": "^16.11.7",
```

**Review by phated — APPROVED** [review]
Excellent. Thanks! I had a few comments but I'm just going to accept the code suggestions and merge.

**Comment by sttk** [comment]
@phated Thank you for your correction and merging!

**Pull request merged**